unetsocket/c/samples/range.c

```diff
@@ -61,6 +62,7 @@ int main(int argc, char *argv[]) {
   // ranging
   rv = unetsocket_get_range(sock, address, range);
   if (rv == 0) printf("Range measured is : %f \n", *range);
+  if (rv != 0) printf("Raging not successful.");
```

Comment:
General comment: Is there a reason to use another `if` instead of `else`?

Reply:
👆🏾 This is applicable across a bunch of samples.

Reply:
Fixed in fe57ea5fb25793077ec4931b35ed04822731fefc